Tests/TestReporting/testSuiteResultParsingSpec.swift

```diff
@@ -9,55 +9,60 @@ class TestSuiteResultParsingSpec: QuickSpec {
             context("when a test log doesn't contain a failure, runtime error, or build error") {
                 it("returns a passed test result") {
                     var contents = loadLogFile(named: "testRunWithoutFailures_withTestSucceededFooter.log")
-                    expect(TestSuiteOutcome.from(testLog: contents)).to(equal(.passed))
+                    expect(TestSuiteOutcome.from(testLog: contents, terminationStatus: 0)).to(equal(.passed))
```

Comment:
I used `0` for everything except the one case I cared about testing. We should probably think about whether any of the rest of these tests need to change or be expanded.

Reply:
I agree (left a comment above b4 reading it, sorry 😄 )